apps/laboratory/src/layout/LayoutHeader.tsx

```diff
@@ -43,6 +46,25 @@ export function LayoutHeader() {
         <Button rightIcon={<IoSettingsOutline />} onClick={controls.onOpen}>
           Options
         </Button>
+        <Button
+          rightIcon={<DownloadIcon />}
+          onClick={() => {
+            ;(
+              window as unknown as { downloadLogsBlobInBrowser?: () => void }
+            ).downloadLogsBlobInBrowser?.()
+            ;(
+              window as unknown as { downloadAppKitLogsBlob: Record<string, () => void> }
+            ).downloadAppKitLogsBlob?.['sdk']?.()
```

Comment:
can we make this a bit cleaner? 
maybe something like 
``` 
type WindowWithLogs = typeof Window & {
 downloadLogsBlobInBrowser?: () => void;
 downloadAppKitLogsBlob: Record<string, () => void>
}
...
function downloadAppkitLogs() {
 const logWindow = window as unknown as WindowWithLogs
.downloadLogsBlobInBrowser?.()
.downloadAppKitLogsBlob?.()
}```

Reply:
Updated in [a70f9f4](https://github.com/WalletConnect/web3modal/pull/2477/commits/a70f9f4303be4eab4485d5be2c98b9263453d3b5)